examples/glfx/webpack.config.js

```diff
@@ -1,4 +1,9 @@
 /* global __dirname, require, module */
+const {resolve} = require('path');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const ALIASES = require('ocular-dev-tools/config/ocular.config')({
```

Comment:
Why does this example need aliases? Does it only work when compiled against source?

Reply:
> Why does this example need aliases? Does it only work when compiled against source?

Yes it is just an HTML file that imports a script with the raw jQuery example code (no import/export), and it first builds a script with the luma.gl symbols exposed as a global.

We now publish some (untested) luma.gl scripts files from the various modules, so could try to use those instead in a later PR.

For now, I just want to make sure this example is working and available as a reference for the ongoing deck.gl effects work.